### PR TITLE
Update feature request template

### PR DESCRIPTION
**Pull request by thechenky**
Issue template updates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/---feature-request.md [hunk 1]
```diff
@@ -5,33 +5,7 @@ about: Have a feature you'd like to see in the functions SDK? Request it through
...
-========================
 Great, we love hearing how we can improve our products! However, GitHub is not the place to submit them. Please submit your feature requests to:
 https://firebase.google.com/support/contact/bugs-features/
-
-Have a usage question?
-=======================
-We get lots of those and we love helping you, but GitHub is not the best place for them and they
-will be closed. Here are some resources to get help:
-
-- Start with the quickstart: https://firebase.google.com/docs/functions/write-firebase-functions
```
Comment (kevinajian) [inline .github/ISSUE_TEMPLATE/---feature-request.md]: Should we move these to the readme then?

Comment (thechenky) [reply .github/ISSUE_TEMPLATE/---feature-request.md]: Sounds good

**Comment by thechenky** [comment]
@kevinajian PTAL

File: README.md [hunk 2]
```diff
@@ -8,6 +8,19 @@ Cloud Functions is a hosted, private, and scalable Node.js environment where you
 
 Learn more about the Firebase SDK for Cloud Functions in the [Firebase documentation](https://firebase.google.com/docs/functions/) or [check out our samples](https://github.com/firebase/functions-samples).
 
+### Have a usage question?
```
[the remark this answers is not in the document]
Comment (thechenky) [reply README.md]: done

**Pull request merged**